docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,292 @@
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage EKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an EKS cluster with Kubernetes version 1.19 or above in order to use our most up-to-date configuration files.
+* Kubernetes node group must be labeled with key as `agentpool` and value as `scalardlpool` for ledger and envoy deployment.
+
+### Recommendations
+
+* Kubernetes node size should be `m5.large` for deploying `ledger` and `envoy` pods, because each node requires 2 vcpu and 8gb ram for deploying ledger and envoy pods.
+* Node groups should have at least 3 nodes for high availability in production.
+
+
+### Create a Kubernetes cluster	
+
+Kubernetes cluster is a primary component of EKS. Scalar DL requires a single EKS cluster for deploying ledger, envoy and monitor services. This section will help you to create an EKS cluster. 
+
+[Create an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) with the above requirements.
+
+> **Tip**
+>
+> If you are creating a Kubernetes cluster on a private subnet, you must add a new rule in the Kubernetes Security Group to enable HTTPS access to the public subnet from the Kubernetes cluster.
+
+### Create managed node groups
+
+This section will help you to create two managed node groups for Scalar DL deployment. 
+Create a managed node group for Ledger and Envoy deployment, and create another managed node group for deploying logs and metrics collection agents.
+
+[Create a managed node group](https://docs.aws.amazon.com/eks/latest/userguide/create-managed-node-group.html) with the above requirements and recommendations.
+
+## Step 4. Install Scalar DL
+
+Scalar DL is a blockchain-inspired tamper-evident distributed ledger software. 
+It executes scalable and highly available ACID-compliant smart contracts with digital signatures in the form of distributed transactions. 
+This section will help you to install Scalar DL to the EKS cluster with [Helm charts](https://github.com/scalar-labs/helm-charts.git).
+
+### Prerequisites
+
+Install the following tools on your host machine:
+
+* [helm](https://helm.sh/docs/intro/install/): helm command-line tool to manage releases in the EKS cluster. In this tutorial, it is used to deploy Scalar DL and Schema loading helm charts to the EKS cluster. Helm version 3.5 or latest is required.
+
+### Requirements
+
+* You must have the authority to pull `scalar-ledger` and `scalardl-schema-loader` container images.
+* You must configure the database properties in the helm chart custom values file.
+* You must confirm that the node group Kubernetes label matches the label in the helm chart [custom values](https://github.com/scalar-labs/scalar-kubernetes/tree/master/conf) file.
+* You must confirm the replica count of the ledger and envoy pods in the `scalardl-custom-values.yaml` file based on the number of nodes in a Scalar DL node group.
```

Comment:
What needs to be confirmed?

Reply:
> What needs to be confirmed?
Confirm 3 nodes for 3 envoy and 3 ledger replicas.
Added as example for easy understanding.